### DISCUSSION TITLE
Please tell me your hometown

### DISCUSSION THREAD

**Discussion by t-matsumoto4758** [NAIST-SE/AttractingContributors#194] [General]
https://github.com/t-matsumoto4758/AttractingContributors-YourHometown

**Comment by Gavin-23** [comment]
Added!
Please help me too~~
Thanks 😊
https://github.com/Gavin-23/AttractingContributors-Music

**Reply by t-matsumoto4758** [reply]
Thanks!
I added my favorite song to your list.

**Comment by TakumiHiraide** [comment]
I sended to you a pull request!! Please help me!!
https://github.com/TakumiHiraide/Where-do-you-want-to-go-
